app/Kconfig

```diff
@@ -568,6 +568,15 @@ if SETTINGS
 config ZMK_SETTINGS_RESET_ON_START
     bool "Delete all persistent settings when the keyboard boots"
 
+if ZMK_SETTINGS_RESET_ON_START
+
+config ZMK_SETTINGS_RESET_ON_START_INIT_PRIORITY
+    int "Settings Reset ON Start Initialization Priority"
+    default 60
+
```

Comment:
Might be useful to add a `help` message that indicates it must be greater than `FLASH_INIT_PRIORITY` when settings are stored in flash.

Reply:
Added a help field with a description. LMK what you think of the content. Thanks!

Reply:
LGTM.